src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -542,11 +542,15 @@ def load_arguments(self, _):
                    help='Enable terminate notification')
         c.argument('ultra_ssd_enabled', ultra_ssd_enabled_type)
         c.argument('scale_in_policy', scale_in_policy_type)
+        c.argument('enable_automatic_repairs', min_api='2018-10-01', arg_type=get_three_state_flag(),
```

Comment:
Shall we use `action='store_true'` instead of `get_three_state_flag`?

Reply:
Maybe `get_three_state_flag` is also fine. `automaticRepairsPolicy` is a property of the resource, and `enable_automatic_repairs` could be considered as the `enabled` property of `automaticRepairsPolicy`.

Reply:
I suggest to put `enable_automatic_repairs` and `automatic_repairs_grace_period` in an argument group so customers know that these two arguments need to be used together.

Reply:
yes, they are in the same argument group. `enable_automatic_repairs` and `automatic_repairs_grace_period` are in `az vmss update`. While only `automatic_repairs_grace_period` in `az vmss create`. You can reference above discussion with Catherine, I explain why we design like this. And want to hear your options. Thanks

Reply:
I mean you can add something like `arg_group='Automatic Repair Policy'`, otherwise the two options are apart from each other in alpabetic order in help message. Usually we name the second level property with a prefix of first level property, they will be showed together in help message, for instance, the names could be `automatic_repairs_policy_enable`, `automatic_reapirs_policy_grace_period`, but the first name looks weird with `enable` as suffix, so I think arg_group can work well in this case.

Reply:
arg_group added